app/search/search.js

```diff
@@ -1,6 +1,6 @@
 const sequelize = require('../config/db')
 const { Op } = require('sequelize')
-const { thresholds, maxResults } = require('../constants/search')
+const { thresholds /* , maxResults */ } = require('../constants/search')
```

Comment:
probably easier to duplicate line and comment out - for future reference, but ok for now